src/FSharpSource.targets

```diff
@@ -246,6 +247,19 @@
     <PackageTags       Condition="'$(PackageTags)' == ''"      >Visual F# Compiler FSharp coreclr functional programming</PackageTags>
   </PropertyGroup>
 
+  <Import Project="$(NugetLocalPackagesDir)\Microsoft.Packaging.Tools.1.0.0-preview2-25401-01\build\Microsoft.Packaging.Tools.targets" Condition="Exists('$(NugetLocalPackagesDir)\Microsoft.Packaging.Tools.1.0.0-preview2-25401-01\build\Microsoft.Packaging.Tools.targets')" />
```

Comment:
It could be we don't need all this and would be better off with a simple direct reference to the appropriate netstandard.dll